openslides_backend/shared/patterns.py

```diff
@@ -74,10 +74,6 @@ def collectionfield_from_collection_and_field(collection: str, field: str) -> st
     return f"{collection}{KEYSEPARATOR}{field}"
 
 
-def collectionfield_from_fqid_and_field(fqid: str, field: str) -> str:
```

Comment:
Why do you want t o remove these functions? They are shared/identical with the same set from datastore.

Reply:
Because they are not used and add to missing tests of the code coverage.

Reply:
Then please write a small unit test for them instead of removing them. Removing features to increase coverage is the wrong way ;)

Reply:
I missed this approving the PR and agree strongly with Joshua

Reply:
Okay, I have readded the patterns functions and add easy unittests for them.